02-peer-protocol.md

```diff
@@ -2207,11 +2217,13 @@ A node:
   - if `next_commitment_number` is 1 in both the `channel_reestablish` it
   sent and received:
     - MUST retransmit `channel_ready`.
+    - MUST retransmit `tx_signatures` if it is using channel establishment v2.
```

Comment:
I'm not sure about this retransmit requirement yet.  Can we remove it from this PR and discuss on https://github.com/lightning/bolts/pull/851/files?

Reply:
`tx_signature` retransmission is necessary b/c you don't have confirmation that the peer received them, no? There's no subsequent "ack" like msg that'd indicate that the sigs have been received by the peer (except the `channel_locked` which isn't sent until later).

resending `tx_sigs` is robust against network failures/disconnects.

Reply:
Yep, makes sense.